conductor-client/src/main/java/conductor/Conductor.kt

```diff
@@ -71,6 +74,18 @@ class Conductor(private val driver: Driver) : AutoCloseable {
         }
     }
 
+    private fun waitUntilVisible(element: UiElement) {
+        repeat(10) {
+            if (!ViewUtils.isVisible(viewHierarchy(), element.treeNode)) {
+                LOGGER.info("Element is not visible yet. Waiting.")
+                Thread.sleep(1000)
```

Comment:
nit : could use coroutines here to avoid blocking the thread

Reply:
We would only benefit from that if the rest of Conductor code would use the same coroutine scope everywhere. As it stands, our thread usage is pretty minimal, so this won't give us any gains